### ISSUE TITLE
Update guides and screenshots to match the new menu structure

### ISSUE THREAD

**Issue by mattab** [matomo-org/matomo#7269]
The goal of this issue is to update docs and FAQs to make them correct with our new menu structure (with both the user menu and the `Administration` menu merged into one after #6989 then in https://github.com/piwik/piwik/issues/9354).

In particular:
- Updates user guides mentioning menu
- Update FAQ mentioning menu
- Update screenshots of plugins where the functionality is accessed through menus
  - for example CustomAlerts (we got this feedback today: `I installed Custom Alerts form the Marketplace, but have no idea where to create these alerts such as seen in the screenshots.`)
- Update guides and screenshots from the [developer website](http://developer.piwik.org/)

**Comment by mattab** [comment]
This cannot be postponed after 3.0.0 - having up to date screenshots in our user guides is very important for a good user experience. Once we have left menu in Piwik, our design will be more or less stable a while, and it will be a great time to update all user guides screenshots. Adding to 3.0.0!

**Comment by mattab** [comment]
This should be mostly done now, closing.
